docs/rule/element-attributes-order.md

```diff
@@ -0,0 +1,38 @@
+## element-attributes-order
+
+Element attributes must be ordered as:
+
+* arguments
+* attributes
+* element modifiers
+* ...attributes
+* ?attributes
+
+### Examples
+
+This rule **forbids** the following:
+
+```hbs
+<MyComponent ...attributes @name="Hello" />
+```
+
+```hbs
+<MyComponent {{did-render this.someAction}} @name="Hello" />
+```
+
+```hbs
+<MyComponent ...attributes {{did-render this.someAction}} />
+```
+
+This rule **allows** the following:
+
+```hbs
+<MyComponent @name="1" bar="baz" {{did-render this.someAction}} ...attributes aria-role="button" />
```

Comment:
this does not seem to match the "must be ordered as" order above 🤔

Reply:
order - https://github.com/ember-template-lint/ember-template-lint/issues/684#issuecomment-480830414